corehq/apps/hqwebapp/templates/hqwebapp/bootstrap5/base_paginated_crud.html

```diff
@@ -3,13 +3,6 @@
 {% load hq_shared_tags %}
 {% load crispy_forms_tags %}
 
-{% block js %}
-  {% if not requirejs_main %}
```

Comment:
Verified that all templates that directly inherit from this one use requirejs, and there are no templates inheriting from those templates.

Reply:
@orangejenny are those all templates that inherit from the bootstrap3 version? I don't think there are a lot of templates that inherit from the bootstrap5 version right now. just want to double check...

Reply:
I didn't even notice that template was split. Yeah, nothing inherits from the B5 version, just the B3. I added b7ebeda8fb0 to remove this code from the B3 version.